### PR TITLE
Maintenance update

### LINKED ISSUE #6: Remove `multipoint` file
## Type of issue
What types of issue is it?
_Select the appropriate type(s) that describe this issue_

- Refactor

## Description
As far as I understand, all functionality in `multipoint` have been replaced by `multipointSparse` and is therefore deprecated. If that is the case, then we should remove the file entirely.

### LINKED ISSUE #7: Add Travis

### PR DESCRIPTION
**Pull request by ewu63**
## Purpose
This PR does the following (long) list of things:
- black and flake8 updates
- removed deprecated `MultiPoint` class (closes #6)
- removed deprecated tests, and added a very basic test as a starting point (related #8)
- moved utility functions into a `utils.py` file
- renamed numpy to `np`
- added Travis (closes #7)

**This PR will also officially remove python 2 support.** A new minor version will be released after this is merged.

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- Code style update (formatting, renaming)
- Refactoring (no functional changes, no API changes)

## Testing
The previous tests did not work since it was essentially a MACH runscript. I have added a very basic test, but more testing need to be added, particularly unit tests.

I also tested that this works with the multipoint optimization in the MACH tutorial.

## Checklist
_Put an `x` in the boxes that apply._

- [x] I have run unit and regression tests which pass locally with my changes
- [x] I have added new tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

**Review by joanibal — DISMISSED** [review]

**Comment by joanibal** [comment]
Nice work. There is quite a lot of work here. Should we bump the minor version?

**Comment by ewu63** [comment]
Yes good call, I'll add a commit to update the version.

File: multipoint/multiPoint.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def addProcessorSet(self, setName, nMembers, memberSizes):
             memberSizes = numpy.ones(nMembers) * memberSizes[0]
         else:
             if len(memberSizes) != nMembers:
-                print("Error: The suppliled memberSizes list is not the correct length")
+                print("Error: The suppllied memberSizes list is not the correct length")
```
Comment (marcomangano) [inline multipoint/multiPoint.py]: typo here

File: multipoint/multiPointSparse.py [hunk 2]
```diff
@@ -163,7 +163,7 @@ def addProcessorSet(self, setName, nMembers, memberSizes):
                 memberSizes = numpy.ones(nMembers) * memberSizes[0]
             else:
                 if len(memberSizes) != nMembers:
-                    raise MPError("The suppliled memberSizes list is not the correct length.")
+                    raise MPError("The suppllied memberSizes list is not the correct length.")
```
Comment (marcomangano) [inline multipoint/multiPointSparse.py:167]: same here

File: README.md [hunk 3]
```diff
@@ -1,18 +1,14 @@
...
 Please see the [documentation](https://mdolab-multipoint.readthedocs-hosted.com) for API documentation.
 
 To locally build the documentation, enter the `doc` folder and enter `make html` in terminal.
 You can then view the built documentation in the `_build` folder.
 
-License
--------
-
+## License
 Copyright 2019 MDO Lab. See the LICENSE file for details.
```
Comment (marcomangano) [inline README.md]: Fix the year ;)

**Review by marcomangano — COMMENTED** [review]
Very nice indeed! Check the few typos I pointed out, and then we are ready to merge and tag the new release.

File: multipoint/multiPoint.py (hunk 1 above)
Comment (ewu63) [reply multipoint/multiPoint.py]: I think you're reading an older commit or something, this file has been removed. Could you double check the rest to make sure your review didn't miss any code changes?

Comment (marcomangano) [reply multipoint/multiPoint.py]: You are right, I added this comment following the commit history but I did not check the final state of the code - and missed the `outdated` label when checking by review. Everything else is fine.

File: multipoint/multiPointSparse.py (hunk 2 above)
Comment (ewu63) [reply multipoint/multiPointSparse.py:167]: This is also fixed in a subsequent commit I believe, so the only thing I'll update is the year.

Comment (ewu63) [reply multipoint/multiPointSparse.py:167]: Oh no you're right, sorry. Will fix this

**Review by marcomangano — APPROVED** [review]

**Review by joanibal — APPROVED** [review]
Still good

**Pull request merged**